src/Common/tests/System/Net/VirtualNetwork/VirtualNetwork.cs

```diff
@@ -26,7 +26,13 @@ public class VirtualNetworkConnectionBroken : Exception
         public bool DisableConnectionBreaking { get; set; } = false;
         private bool _connectionBroken = false;
 
-        public void ReadFrame(bool server, out byte[] buffer)
+        public byte[] ReadFrame(bool server) =>
```

Comment:
I wonder how many tests end up calling synchronous ReadFrame. I hope not many since this would still block. 
Can't be worse than before though, where it was 100%  :-)